client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/utils/style-variations.ts

```diff
@@ -14,3 +15,18 @@ export function removeLegacyDesignVariations( designs: Design[] ) {
 		);
 	return designs.filter( ( design ) => ! isVariationOfAnotherDesign( design ) );
 }
+
+/**
+ * Promote designs with style variations by having them at the beginning of the list.
+ */
+export function promoteDesignVariations( designs: Design[] ) {
+	const designsWithVariations = designs.filter(
+		( design ) => design.style_variations && design.style_variations.length > 0
+	);
+
+	const designsWithoutVariations = designs.filter(
+		( design ) => design.style_variations && design.style_variations.length === 0
```

Comment:
```suggestion
		( design ) => ! design.style_variations || design.style_variations.length === 0
```

Reply:
(basically this is just a de morgan negation of the above `designsWithoutVariations` predicate)